app/tools/android/make_apk.py

```diff
@@ -317,9 +317,10 @@ def CopyCompressedLibrary(native_path, library_path, raw_path, arch):
   # need dummy library file to keep the arch info if enable lzma.
   arch_path = os.path.join(library_path, arch)
   if (not os.path.isdir(arch_path)):
-      os.mkdir(arch_path)
+    os.mkdir(arch_path)
   dummy_library = os.path.join(native_path, DUMMY_LIBRARY);
-  shutil.copy(dummy_library, arch_path)
+  if (os.path.isfile(dummy_library)):
+    shutil.copy(dummy_library, arch_path)
```

Comment:
If `dummy_library` is the only thing that you are copying into `arch_path`, doesn't it make more sense to also conditionally create the directory only if the file is there?

```python
arch_path = ...
dummy_library = ...
if os.path.isfile(dummy_library):
  if not os.path.isdir(arch_path)):
    os.mkdir(arch_path)
  shutil.copy(dummy_library, arch_path)
```